frontend-app/src/Thermometer/TemperatureControl.jsx

```diff
@@ -1,14 +1,21 @@
-import { useState } from "react";
+import { useContext, useState } from "react";
+import MessageContext from "../context/MessageContext";
 import './TemperatureControl.css'
 
 function TemperatureControl() {
-  const [temp, setTemp] = useState(10);
+
+  const [temp, setTemp] = useState(0);
+  const myCtx = useContext(MessageContext);
+
 
   const rise = () => {
     temp < 30 && setTemp(temp + 1);
+    myCtx.success('The temperature has increased!');
```

Comment:
вот тут есть одно но
если температура будет 30 и ты нажмешь на +, то появится сообщение, а температура не поднимется
```suggestion
    if (temp < 30) {
        setTemp(temp + 1);
        myCtx.success('The temperature has increased!');
    } 
```

в drop тоже самое